pkg/services/sqlstore/search_builder.go

```diff
@@ -92,8 +94,17 @@ func (sb *SearchBuilder) ToSql() (string, []interface{}) {
 		LEFT OUTER JOIN dashboard folder on folder.id = dashboard.folder_id
 		LEFT OUTER JOIN dashboard_tag on dashboard.id = dashboard_tag.dashboard_id`)
 
-	sb.sql.WriteString(" ORDER BY dashboard.title ASC" + dialect.Limit(5000))
+	// Default to page 1
+	if sb.page < 1 {
+		sb.page = 1
+	}
+
+	// do not allow limits beyond 5000
+	if sb.limit > 5000 {
+		sb.limit = 5000
+	}
 
+	sb.sql.WriteString(" ORDER BY dashboard.title ASC " + dialect.LimitOffset(sb.limit, (sb.page-1)*sb.limit))
```

Comment:
Think we should use `ORDER BY dashboard.title ASC, dashboard.id ASC` in case there are dashboards with the same title in the breakpoint between pages.